setup.py

```diff
@@ -26,7 +26,7 @@ def _get_requirements_from_files(groups_files):
     long_description = f.read()
     
 setup(name='splot', #name of package
-      version='1.1.2',
+      version='1.1.3',
```

Comment:
This is not a critical issue, but instead of hardcoding the name and version number in here, how about adding the following to the top of `setup.py`:
```python
package = "splot"
# Get __version__ from package/__init__.py
with open(package + "/__init__.py", "r") as f:
    exec(f.readline())
```
then replacing the hardcoded name and version parameters in `setup()` with the declared variables:
```python
setup(
    name=package,
    version=__version__,
```

See `spaghetti`'s [`setup.py`](https://github.com/pysal/spaghetti/blob/master/setup.py) as an example.

Reply:
I can actually simply go ahead and do that right now. I'll push a fix to this branch.